content/common/cli-reference.md

```diff
@@ -1208,3 +1209,28 @@ TRIGGER OPTIONS:
 {{ blockEnd }}
 
 {{ blockEnd }}
+
+### CLI Stability Table
+
+CLI commands have three phases that indicate levels of stability:
+
+- `experimental`: These commands are experiments and may or may not be available in later versions of the CLI.
+- `stabilizing`: These commands have moved out of the `experimental` phase and we are now in the active process of stabilizing them. This includes updating flags, command output, errors, and more.
+- `stable`: These commands have moved out of the `stablizing` phase and will not change until the next major version release.
+
+| Command                         | Stability    |
+| ------------------------------- | ------------ |
+| `spin add`                      | Stable       |
+| `spin build`                    | Stable       |
+| `spin new`                      | Stable       |
+| `spin plugins`                  | Stable       |
+| `spin templates`                | Stable       |
+| `spin up`                       | Stable       |
+| `spin cloud deploy`             | Stabilizing  |
+| `spin cloud login`              | Stabilizing  |
+| `spin registry`                 | Stabilizing  |
+| `spin bindle <all subcommands>` | Experimental |
```

Comment:
`<all subcommands>` isn't stated on `spin plugins` and `spin templates` - what is our preferred convention?

Reply:
I think we should list all commands since that list might change over time. Better to be explicit.

`spin bindle <prepare | push>`
`spin plugins <list | uninstall...etc>`

Reply:
That's certainly the safest approach.  There's a risk it could bulk out the table (commands like deploy and up getting lost in all the plugin and template ones) but we should definitely try it that way, we can always iterate if we don't like it!

Reply:
updated.